calligraphy/src/main/java/io/github/inflationx/calligraphy3/TypefaceUtils.java

```diff
@@ -32,7 +33,13 @@ public static Typeface load(final AssetManager assetManager, final String filePa
         synchronized (sCachedFonts) {
             try {
                 if (!sCachedFonts.containsKey(filePath)) {
-                    final Typeface typeface = Typeface.createFromAsset(assetManager, filePath);
+                    final File file = new File(filePath);
+                    final Typeface typeface;
+                    if(file.isAbsolute()) {
```

Comment:
better yet, let's just skip the `File` instance and check the first path char. Per the doc:

> On Android, absolute paths start with the character '/'.

Reply:
```java
                    if (filePath.startsWith("/")) {
                        // Absolute path, so load from that file path
                        typeface = Typeface.createFromFile(new File(filePath));
                    } else {
                        typeface = Typeface.createFromAsset(assetManager, filePath);
                    }
```